### ISSUE TITLE
Log file of failed downloads

### ISSUE THREAD

**Issue by olegafx** [dgorissen/coursera-dl#4]
Now it's difficult to see which materials wasn't downloaded because of some error. It's better lo write all of the failed downloads to a file.

**Comment by dgorissen** [comment]
yeah, as well as switching to python logging. Time is at a premium for me at the moment but will have a look at it. Pull requests welcome though :). 

Edit: By the way a simple fix is just to redirect the script output through grep

**Comment by mattmcgiv** [comment]
If this is still an issue, I would like to take it on and submit a pull request.

**Comment by dgorissen** [comment]
Yes the switch to using logging (and associated log levels) would be good. All contributions welcome.
